Core/include/Acts/EventData/SubspaceHelpers.hpp

```diff
@@ -69,7 +66,8 @@ inline static SerializedSubspaceIndices serializeSubspaceIndices(
 {
   SerializedSubspaceIndices result = 0;
   for (std::size_t i = 0; i < FullSize; ++i) {
-    result |= static_cast<SerializedSubspaceIndices>(indices[i]) << (i * 8);
+    result |= static_cast<SerializedSubspaceIndices>(indices[i] & 0xFF)
```

Comment:
What does the extra `& 0xFF` do?

Reply:
I thought it might be a good idea to cut out the 8 bits and then shift them. In case there is any other bit set it would lead to weird results

Reply:
Ah I didn't realize this cuts to 8 bits. Fair enough.